packages/yoast/README.md

```diff
@@ -40,6 +40,8 @@ export default {
 };
 ```
 
+> Comprehensive information about how to use the @frontity/yoast package can be found in the [docs](https://api.frontity.org/frontity-packages/features-packages/yoast.md).
```

Comment:
Should the link not have the `.md` extension? 🤔 
```suggestion
> Comprehensive information about how to use the @frontity/yoast package can be found in the [docs](https://api.frontity.org/frontity-packages/features-packages/yoast).
```

Reply:
You're right - I've fixed it.